teamB/cheonsongin/models/likeDao.js

```diff
@@ -0,0 +1,40 @@
+const { DataSource } = require('typeorm');
+
+const appDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+});
+
+appDataSource.initialize()
+  .then(() => {
+    console.log('Data Source has been initialized!');
+  })
+  .catch((err) => {
+    console.error('Error occurred during Data Source initialization', err);
+    appDataSource.destroy();
+  });
```

Comment:
해당 코드는 반복이 되고 있습니다.

지금은 3개의 파일만 관리하면 되므로 유지보수 측면에서 어려움이 없지만 100개가 된다면 굉장히 비효율적이고 불필요한 코드일 것입니다.

models 파일 혹은 utils 파일 내에 `dataSource.js` 혹은 `appDataSource.js` 파일 생성 후 해당 코드를 옮겨적어주세요.
이후 DataSource라는 새로운 객체를 담은 appDataSource 객체를 내보내주면 됩니다!

```suggestion
// 모듈 분리
const { DataSource } = require('typeorm');

const appDataSource = new DataSource({
  type: process.env.TYPEORM_CONNECTION,
  host: process.env.TYPEORM_HOST,
  port: process.env.TYPEORM_PORT,
  username: process.env.TYPEORM_USERNAME,
  password: process.env.TYPEORM_PASSWORD,
  database: process.env.TYPEORM_DATABASE
});

appDataSource.initialize()
  .then(() => {
    console.log('Data Source has been initialized!');
  })
  .catch((err) => {
    console.error('Error occurred during Data Source initialization', err);
    appDataSource.destroy();
  });
  
  module.exports = { appDataSource }
```